lib/connectors/base/extractor.rb

```diff
@@ -12,8 +12,8 @@
 require 'date'
 
 require 'utility'
+require 'stubs/app_config'
 require 'stubs/connectors/stats' unless defined?(Rails)
-require 'stubs/connectors' unless defined?(Rails)
```

Comment:
When will `defined?(Rails)` be `true`?

Reply:
It never will be, it was done so that this file could be used in Rails apps as a part of a gem. At this point it's safe to remove this everywhere.